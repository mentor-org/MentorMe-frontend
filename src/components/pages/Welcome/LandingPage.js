import React from 'react';
import { Link } from 'react-router-dom';


const LandingPage = () => {
  return (
    <div className="container home">
      <div className="d-flex">
        <div className="half">
          <Link to="/" className="logo">
            <h3>MentorMe</h3>
          </Link>
        </div>
        <div className="half">
          <div className="align-item">
            <Link to="/auth/signup" className="btn">Get Started</Link>
          </div>
        </div>
      </div>
    </div>
  );
};

export default LandingPage;
