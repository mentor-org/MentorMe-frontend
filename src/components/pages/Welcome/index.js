import React from 'react';

const Welcome = () => {
  return (
    <div className="container home"></div>
  );
};

export default Welcome;
