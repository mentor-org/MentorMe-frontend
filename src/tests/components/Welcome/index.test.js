import React from 'react';
import Welcome from '../../../components/pages/Welcome';

describe('App', () => {
  it('should render the App component correctly', () => {
      const shallowWrapper = shallow(<Welcome />);

      expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
