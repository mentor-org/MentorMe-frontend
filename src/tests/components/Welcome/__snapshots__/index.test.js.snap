// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App component correctly 1`] = `
<div
  className="container home"
/>
`;
